aspnetcore/blazor/hybrid/tutorials/wpf.md

```diff
@@ -64,6 +64,19 @@ At the top of the project file, change the SDK to `Microsoft.NET.Sdk.Razor`:
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 ```
 
+:::moniker range=">= aspnetcore-10.0"
+
+In the project file's existing `<TargetFramework>` property, add a Windows 10 version number:
+
+```xml
+<TargetFramework>net10.0-windows10.0.17763.0</TargetFramework>
```

Comment:
I don't think that we're headed in the right direction here. Creating a project from the .NET 10 template yields a .NET 10 TFM for Windows ...

```xml
<TargetFramework>net10.0-windows</TargetFramework>
```

... and the article assumes the current .NET release is used (or that the dev selected a different release).

There's no need for this PR under the current conventions that we use for coverage.